charts/manifests/templates/utils/_common.tpl

```diff
@@ -49,3 +49,19 @@ limitations under the License.
     {{- fail "Template requires '.context' as arguments" }}
   {{- end }}
 {{- end -}}
+
+{{/*
+  Sprig Template - selectorLabels Template Wrapper. Uses the Bundlename as service selector if no other selector is defined.
+*/}}
+{{- define "bedag-lib.utils.common.selectorLabels" -}}
+  {{- if and $.Values.selectorLabels (kindIs "map" $.Values.selectorLabels) }}
+    {{- include "lib.utils.strings.template" (dict "value" $.Values.selectorLabels "context" $) | indent 0 }}
+  {{- else }}
```

Comment:
What's that for?

Reply:
that was to prevent custom defined labels from being overwritten, which apparently doesn't happen, will delete in next revision.